Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/MotionController/AttachToController.cs

```diff
@@ -1,47 +1,37 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
-using Microsoft.MixedReality.Toolkit.Internal.Attributes;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Utilities;
 using Microsoft.MixedReality.Toolkit.Internal.EventDatum.Input;
 using Microsoft.MixedReality.Toolkit.Internal.Extensions;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
 using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
 using Microsoft.MixedReality.Toolkit.SDK.Input;
 using UnityEngine;
 
 namespace Microsoft.MixedReality.Toolkit.SDK.UX.MotionController
 {
     /// <summary>
-    /// Waits for a controller to be initialized, then attaches itself to a specified element
+    /// Waits for a controller to be initialized, then attaches itself to a specified hand.
     /// </summary>
     public class AttachToController : InputSystemGlobalListener, IMixedRealitySourcePoseHandler
     {
         [SerializeField]
-        protected Vector3 PositionOffset = Vector3.zero;
```

Comment:
Why are these removed now? These are offsets from the controller, not offsets for the controller. Where's that now handled by the visualizer?

Reply:
Where? It's a massive PR. The only thing I see are controller offsets.